src/styles/components/Select.scss

```diff
@@ -0,0 +1,77 @@
+@import "../common/fonts";
```

Comment:
Copyright header is missing in this file

Reply:
done